scripts/plot_dataframe.py

```diff
@@ -87,7 +83,7 @@ def plot_analyzed_dataframe(args: Namespace) -> None:
     )
 
     df_buy = data[data['buy'] == 1]
-    buys = go.Scattergl(
+    buys = go.Scatter(
```

Comment:
Why this change? It makes the whole graph drastically slower by removing the OpenGL acceleration.

Reply:
On my Pc ( Win10 ), opening this file will use by default Edge, and the buy / sell graph was not displayed at all. 
Hence this change.
This has probably happened before, because the other graphs are rendered without WebGL ( like the candle and the Bollinger bands ).
So I think the impact on perf for just the few buy/ sell items is negligeable ( and it adds more compatibility )